rstudio/rstudio
# remove obsolete useinternet2 preference

### Intent

Addresses #13250

### Approach

This preference was used to control passing `--internet2` to R, but this option is no longer needed as it is the default in all supported versions of R.

Removed the preference and the UI for setting it, along with associated strings and logic that acted on the pref.

### Automated Tests

None

### QA Notes

Confirm the preference no longer shows up in Global Options / Packages on Windows Desktop, and you can still download packages and check for RStudio updates.

### Documentation

None

### Checklist
